client/src/GamemasterInGame/NightOrder.tsx

```diff
@@ -1,34 +1,31 @@
-import { useState } from "react";
-import { useDefiniteGame } from "../store/GameContext";
+import React from "react";
 import { getRole, getRoleExtension } from "../assets/game_data/gameData";
-import { RoleIcon, RoleName, RoleText } from "../shared/RoleIcon";
-import { Button, Checkbox, Flex, IconButton } from "@radix-ui/themes";
-import { MeaningfulIcon } from "../PlayerLanding/PlayerWaiting/MeaningfulIcon";
-import { useDecideFate } from "../store/useStore";
-import { GiRaiseZombie } from "react-icons/gi";
-import { PiKnifeBold } from "react-icons/pi";
-import classNames from "classnames";
+import { IngamePlayerList } from "./PlayerList";
+import { Button, Flex } from "@radix-ui/themes";
+import { useDefiniteGame } from "../store/GameContext";
+
+// interface NightOrderProps {
+//   playersToRoles: Record<string, Role>;
+//   deadPlayers: Record<string, boolean>;
+// }
 
 export function NightOrder() {
   const { game } = useDefiniteGame();
```

Comment:
I'd tried passing the above interface props in, but playerToRoles was always null for whatever reason

Reply:
dunno what you were foing wrong.  But I like not passing props and letting components control things